src/Avalonia.Visuals/Rendering/SceneGraph/LineBoundsHelper.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using Avalonia.Media;
+
+namespace Avalonia.Rendering.SceneGraph
+{
+    public static class LineBoundsHelper
+    {
+        private const double degreeToRadians = Math.PI / 180.0;
+
+        private static double CalculateAngle(Point p1, Point p2)
+        {
+            var xDiff = p2.X - p1.X;
+            var yDiff = p2.Y - p1.Y;
+
+            return Math.Atan2(yDiff, xDiff) * 180.0 / Math.PI;
```

Comment:
if my math skills hasnt rusted, Atan2 already returns radians, `180/Math.Pi` converts rads to degrees so if you remove this you can remove the `degreeToRadians` stuff in other functions